assets/prototype/domain/shared/predicates/tickets/filters/percentSoldAtOrAbove.ts

```diff
@@ -0,0 +1,24 @@
+import validFiniteQuantity from './validFiniteQuantity';
+
+type PercentSoldAtOrAboveProps = {
+	maxQuantity: number;
+	ticketEntities: any[];
+};
+
+/**
+ * @param {Object} ticketEntity    event ticket object
+ * @param {number} maxQuantity
+ * @return {boolean} true if sold/qty >= maxQuantity
+ */
+const percentSoldAtOrAbove = ({ maxQuantity, ticketEntities }: PercentSoldAtOrAboveProps) => {
+	const filterFn = (ticketEntity) => {
+		return (
+			validFiniteQuantity(ticketEntity) &&
+			parseInt(ticketEntity.sold, 10) / parseInt(ticketEntity.qty, 10) >= maxQuantity / 100
```

Comment:
Same here, not :100: happy with these.
I need to come back on the next iteration and rethink this.